cli/flags.rs

```diff
@@ -509,6 +515,23 @@ fn resolve_paths(paths: Vec<String>) -> Vec<String> {
   out
 }
 
+fn resolve_urls(urls: Vec<String>) -> Vec<String> {
+  let mut out: Vec<String> = vec![];
+  for urlstr in urls.iter() {
+    let result = Url::from_str(urlstr);
+    if result.is_err() {
+      eprintln!("Bad Url: {}", urlstr);
```

Comment:
Error instead of continuing

Reply:
maybe in file whitelist also in that case?